packages/react-aria-components/src/GridList.tsx

```diff
@@ -622,7 +622,7 @@ export const GridListHeader = /*#__PURE__*/ createLeafComponent(HeaderNode, func
   let rowHeaderProps = useContext(GridListHeaderContext);
 
   return (
-    <header {...props} ref={ref}>
+    <header {...props} ref={ref} className="react-aria-GridListHeader">
```

Comment:
i forgot to add this in the last PR oops 😅

Reply:
Put the className before the props spread so that people can override it.